client/src/components/Login.jsx

```diff
@@ -0,0 +1,109 @@
+import React, {useState, useEffect} from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+
+const Login = (props) => {
+
+  const [userName, setUserName] = useState('');
+  const [password, setPassword] = useState('');
+
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    let query = {userName, password};
+    axios.post('/api/login', query)
+    .then((result) => {
+      if(result.data === false) {alert('Login Failed')}
+      else {
+      console.log('this is results.data in Login: ', result.data)
+      let sendUser = async () => {
+        props.setLoginUserName(result.data)
+      }
+      sendUser().then(props.setIsLoggedIn(true));
+      }
+    })
+    setUserName('');
+    setPassword('');
+  }
+
+  const handleNewUser = (e) => {
+    e.preventDefault();
+    let query = {userName, password};
+    axios.post('/api/addUser', query)
+    .then((result) => {
+      // console.log('this is result: ', result)
+      props.setIsLoggedIn(true);
+    })
+    setUserName('');
+    setPassword('');
+  }
+
+  return (
+    <LoginStyle>
+    <Title> New Wiz WhoDis?</Title>
+    <form>
+    <Div>
+      <Label>What is your Wizard Name?</Label>
+    </Div>
+  <Text type="text" color="red" id="userName" value={userName} onChange={e => setUserName(e.target.value)} name="userName"></Text>
+  <Div>
+      <Label>What is your Secret Spell?</Label>
+    </Div>
+  <Text type="text" color='antiquewhite' id="password" value={password} onChange={e => setPassword(e.target.value)} name="password"></Text>
```

Comment:
Does this show the password in plaintext while the user types? You can change the form element to properly obscure the password text.